src/redux/actions/auth.js

```diff
@@ -0,0 +1,40 @@
+import * as ACTIONS from 'constants/action_types';
+import Lbryio from 'lbryio';
+
+export function doNewInstallation(installationId) {
+  return dispatch => {
+    dispatch({
+      type: ACTIONS.GET_AUTH_TOKEN_STARTED,
+    });
+
+    Lbryio.call(
+      'user',
+      'new',
+      {
+        auth_token: '',
+        language: 'en',
+        app_id: installationId,
+      },
+      'post'
+    )
+      .then(response => {
+        if (!response.auth_token) {
+          dispatch({
+            type: ACTIONS.GET_AUTH_TOKEN_COMPLETED,
+            data: { authToken: null },
+          });
+        } else {
+          dispatch({
+            type: ACTIONS.GET_AUTH_TOKEN_COMPLETED,
+            data: { authToken: response.auth_token },
+          });
+        }
+      })
+      .catch(() => {
+        dispatch({
+          type: ACTIONS.GET_AUTH_TOKEN_COMPLETED,
+          data: { authToken: null },
+        });
+      });
+  };
+}
```

Comment:
Guessing you just haven't gotten to it, but worth porting this too https://github.com/lbryio/lbry-app/blob/171ea2b97b6a393c880995a42cd4189cdda3e5bd/src/renderer/redux/actions/user.js#L39